src/gslc/jsonAssembly.fs

```diff
@@ -0,0 +1,89 @@
+﻿module jsonAssembly
+
+open System.IO
+open System.Text
+open System
+open commonTypes
+open constants
+open Amyris.Bio.utils
+open parseTypes
+open shared
+open Newtonsoft.Json
+open System.Collections.Generic
+
+/// A few key fields of a DNA slice for JSON representation.
+type DNASliceJson =
+   {id: string; 
+    extId: string; 
+    dna: string;  
+    sourceChr: string; 
+    sourceFr: string; 
+    sourceTo: string; 
+    sourceFwd: bool;
+    destFr: string; 
+    destTo: string;
+    destFwd: bool; 
+    amplified: bool; 
+    sliceName: string;
+    sliceType: string;
+    breed: string;
+    description: string ; 
+}
+
+/// A few key fields of Assembly output for JSON representation.
+type AssemblyOutJson = 
+    { id: string;
```

Comment:
Please fix this type declaration to have idiomatic indentation, similar to the above DNASliceJson.